viya4-iac-aws-darksite/custom-ami/README.md

```diff
@@ -0,0 +1,159 @@
+## AWS Darksite Custom AMI
+By default, viya4-iac-aws uses the base Canonical Ubuntu Server image, which does not have the required nfs-kernel-server package.  IAC installs these packages as part of the cloud-init process, however this is not possible in a dark site which has no outbound internet access.  Below are instructions for creating a custom AMI which will ensure that nfs-kernel-server packages are installed for both the jumpserver and NFS server.
+
+Once the AMI is created, you can then apply the mod to viya4-iac-aws by running the [darksite-iac-aws-mods.sh](https://gitlab.sas.com/jocobu/viya4-aws-darksite/-/tree/main/viya4-iac-aws/darksite-iac-aws-mods/darksite-iac-aws-mods.sh) script.
+
+## Notes
+This information was provided by [Frederik Vandenberghe](https://gitlab.sas.com/sbxfrv).  
+
+## Launch EC2-instance based on the current AMI in use
+First thing to do is launch an EC2-instance based on the AMI that's in use as AWS-base image for the nfs-server and the jump-server.  You can see which AMI that is in https://github.com/sassoftware/viya4-iac-aws/blob/main/modules/aws_vm/main.tf
+
+![](img/img1.png)
+
+It's an AMI owned by Canonical AWS account (Canonical is company behind Ubuntu) that provides an Ubuntu AWS AMI images.  So you can find that image in the AMI Catalog by searching with the same search criteria as specified in that main.tf file.
+
+![](img/img2.png) ![](img/img3.png)
+
+
+And exactly like in the main.tf (most_recent = true) select the most recent version and click on Launch Instance with AMI.
+
+![](img/img4.png)
+
+
+
+Give the new instance a name that you will remember.  I choosed m5.xlarge as instance type to stay in sync with the iac scripts.
+
+![](img/img5.png)
+
+For the keypair, you'll want to use a keypair you own and the same keypair you plan on using for viya4-iac-aws and viya4-deployment.  If the public key you'd like to use isn't in EC2 already.. import it now (EC2 > key pairs > import key pair).  In the screenshot below, I've already added it named "aws-key".
+
+<img src="img/img6.png" style="zoom:60%;" />
+
+Click launch instance.
+
+![](img/img6b.png)
+
+Wait for the initialization, check for the assigned IP address and setup SSH session to the running EC2-instance with your private SSH-key.  User to use will be ubuntu. 
+
+![](img/img7.png)
+
+
+
+## Install nfs-kernel-server on running EC2-instance
+
+Run the following commands on the Ubuntu EC2-instance.
+
+```shell
+sudo apt update
+
+sudo apt upgrade
+
+sudo apt install nfs-kernel-server
+```
+
+![](img/img8.png)
```

Comment:
If we're providing images for the output of the commands, could those images be associated with the command that was run? Seems like we state the 3 commands and then have the 3 outputs from those commands.